ver/us/splat.yaml

```diff
@@ -4163,7 +4163,6 @@ segments:
     vram: 0xE0120000
     subsegments:
     - [0x40B3F0, c, sun]
-    - [0x40BB70, data]
```

Comment:
for the sake of record keeping/whatever, we tend to leave these as `- [0x40BB70]`